synth/src/cli/telemetry.rs

```diff
@@ -40,7 +101,11 @@ where
     Fut: Future<Output = Result<T, E>>,
     E: AsRef<dyn Error + 'static>
 {
-    let client = TelemetryClient::new();
+    if is_enabled() {
+        let synth_command = serde_json::to_string(&args).unwrap();
```

Comment:
I couldn't find a way to delay this serialization until inside the panic hook, unfortunately.

Reply:
Short of adding `#[derive(Clone)]` to the `Args` and moving a clone of `args` into the closure, I don't see a way to do it either.